modules/core/src/main/java/org/apache/ignite/internal/processors/query/RunningQueryManager.java

```diff
@@ -72,6 +77,18 @@
     /** Query history tracker. */
     private volatile QueryHistoryTracker qryHistTracker;
 
+    /** Number of successfully executed queries. */
+    private final LongAdderMetric successQrsCnt;
```

Comment:
Why LongAdderMetric is used for succeeded queries and AtomicLongMetric is used for failed\canceled ones?